packages/studio/src/components/AddPageButton/EntityIdField.tsx

```diff
@@ -116,6 +114,12 @@ const selectStyles = {
     borderTopLeftRadius: "0",
     borderBottomLeftRadius: "0",
     borderColor: "rgb(156, 163, 175)",
+    borderTopRightRadius: "0.5rem",
+    borderBottomRightRadius: "0.5rem",
+  }),
+  menu: (base) => ({
+    ...base,
+    fontSize: "14px",
```

Comment:
nit: can we also add this `fontSize` to the placeholder text?

Reply:
I passed text-sm to the entire Select